game/domain/maneuver/interact.lua

```diff
@@ -60,7 +60,7 @@ function INTERACT.exhaustionCost(actor, inputvalues)
 end
 
 function INTERACT.validate(actor, inputvalues)
-  return not not _seek(actor, inputvalues)
+  return not not _seek(actor, inputvalues) and not actor:isFocused()
```

Comment:
I'd invert the order for efficiency and because the focus seems like a more important factor than whether or not there is something interactable nearby.